internal/validators/tests/validators-specs.js

```diff
@@ -0,0 +1,126 @@
+import {
+	warn,
+	validateRange,
+	validateRangeOnce,
+	validateStepped
+} from "../validators";
+
+describe("validators", () => {
+	let consoleSpy;
+
+	beforeEach(() => {
+		consoleSpy = jest.spyOn(console, 'warn').mockImplementation(() => {});
+	});
+
+	describe("validateRange", function () {
+		test("should throw a console warning", () => {
+			warn("Warning!");
+
+			expect(consoleSpy).toHaveBeenCalled();
+			expect(consoleSpy.mock.calls).toMatchInlineSnapshot(`
+        Array [
```

Comment:
Please check indentation for the snapshots. (for all the tests below)

Reply:
This indentation format was generated by enact lint -s --fix.